src/main/studio/kdb/Config.java

```diff
@@ -435,6 +438,148 @@ public void save() {
         }
     }
 
+    public Object serverTreeToObj(ServerTreeNode root) {
+        //converts the server tree to an object that can be saved into JSON
+        LinkedHashMap<String,Object> result = new LinkedHashMap<>();
+        result.put("name", root.getName());
```

Comment:
Such encoding assumes that leaf node names are unique which is not guaranteed. I assume import/export logic would be broken if i have two or more severs with the same name but in the different location in the tree

Reply:
In fact server names should be unique otherwise there is a chance that there will be a clash when moving them under the same parent. There was already a feature request to enforce this which I didn't get around to implement yet but looks like I will have to do it now.

Reply:
My view that we need to enforce uniqueness of full name, I know that people already have configuration with the same names under different locations in the server tree. If there are use-cases when uniqueness of full name is not guaranteed, i think they should be fixed.

Reply:
I will add this as a future enhancement since it is too involved to deal with in this PR which is to reapply existing changes, i.e. this code is already being run by users who are not reporting any problems with it.
#81